### ISSUE TITLE
Browser access type Prometheus query does not render time series despite of correct query/response

### ISSUE THREAD

**Issue by simongun** [grafana/grafana#12220]
Please include this information:
### What did you do? 
Use a Prometheus Datasource with access type "Browser"; Prometheus is protected with an Oauth proxy. User is logged in there, in same Browser session. Prometheus returns a time series, but Grafana does not display it; the "eye" symbol in the Graph config also triggers a correct query/response, but also nothing is displayed.
The Query Inspector triggers a query (seen in the network trace) and gets a valid response, but the "spinner" symbol keeps showing and the response is not displayed in the Inspector box.

### What was the expected result?
For Chart to show time series and Inspector show the result
### What happened instead?
No data in chart and no result in Inspector
### Include raw network request & response: get by opening Chrome Dev Tools 
https://prometheus.domain.com/api/v1/query_range?query=hello_worlds_total&start=1528495200&end=1528581600&step=60
 {"status":"success","data":{"resultType":"matrix","result":[{"metric":{"__name__":"hello_worlds_total","app":"sample-app","deployment":"sample-app-2","deploymentconfig":"sample-app","instance":"xxxxxxxxxx:8443","job":"kubernetes-service-endpoints","kubernetes_name":"sample-app","kubernetes_namespace":"simontest1-dev-ats-openpaas-cc","kubernetes_pod_name":"sample-app-2-j955h"},"values":[[1528495200,"10"],[1528495260,"10"],[1528495320,"10"],[1528495380,"10"],[1528495440,"10"],[1528495500,"10"],[1528495560,"10"],[1528495620,"10"],[1528495680,"10"],[1528495740,"10"],[1528495800,"10"],[1528495860,"10"],[1528495920,"10"],[1528495980,"10"],[1528496040,"10"],[1528496100,"10"],[1528496160,"10"],[1528496220,"10"],[1528496280,"10"],[1528496340,"10"],[1528496400,"10"],[1528496460,"10"],[1528496520,"10"],[1528496580,"10"],[1528496640,"10"],[1528496700,"10"],[1528496760,"10"],[1528496820,"10"],[1528496880,"10"],[1528496940,"10"],[1528497000,"10"],[1528497060,"10"],[1528497120,"10"],[1528497180,"10"],[1528497240,"10"],[1528497300,"10"],[1528497360,"10"],[1528497420,"10"],[1528497480,"10"],[1528497540,"10"],[1528497600,"10"],[1528497660,"10"],[1528497720,"10"],[1528497780,"10"],[1528497840,"10"],[1528497900,"10"],[1528497960,"10"],[1528498020,"10"],[1528498080,"10"],[1528498140,"10"],[1528498200,"10"],[1528498260,"10"],[1528498320,"10"],[1528498380,"10"],[1528498440,"10"],[1528498500,"10"],[1528498560,"10"],[1528498620,"10"],[1528498680,"10"],[1528498740,"10"],[1528498800,"10"],[1528498860,"10"],[1528498920,"10"],[1528498980,"10"],[1528499040,"10"],[1528499100,"10"],[1528499160,"10"],[1528499220,"10"],[1528499280,"10"],[1528499340,"10"],[1528499400,"10"],[1528499460,"10"],[1528499520,"10"],[1528499580,"10"],[1528499640,"10"],[1528499700,"10"],[1528499760,"10"],[1528499820,"10"],[1528499880,"10"],[1528499940,"10"],[1528500000,"10"],[1528500060,"10"],[1528500120,"10"],[1528500180,"10"],[1528500240,"10"],[1528500300,"10"],[1528500360,"10"],[1528500420,"10"],[1528500480,"10"],[1528500540,"10"],[1528500600,"10"],[1528500660,"10"],[1528500720,"10"],[1528500780,"10"],[1528500840,"10"],[1528500900,"10"],[1528500960,"10"],[1528501020,"10"],[1528501080,"10"],[1528501140,"10"],[1528501200,"10"],[1528501260,"10"],[1528501320,"10"],[1528501380,"10"],[1528501440,"10"],[1528501500,"10"],[1528501560,"10"],[1528501620,"10"],[1528501680,"10"],[1528501740,"10"],[1528501800,"10"],[1528501860,"10"],[1528501920,"10"],[1528501980,"10"],[1528502040,"10"],[1528502100,"10"],[1528502160,"10"],[1528502220,"10"],[1528502280,"10"],[1528502340,"10"],[1528502400,"10"],[1528502460,"10"],[1528502520,"10"],[1528502580,"10"],[1528502640,"10"],[1528502700,"10"],[1528502760,"10"],[1528502820,"10"],[1528502880,"10"],[1528502940,"10"],[1528503000,"10"],[1528503060,"10"],[1528503120,"10"],[1528503180,"10"],[1528503240,"10"],[1528503300,"10"],[1528503360,"10"],[1528503420,"10"],[1528503480,"10"],[1528503540,"10"],[1528503600,"10"],[1528503660,"10"],[1528503720,"10"],[1528503780,"10"],[1528503840,"10"],[1528503900,"10"],[1528503960,"10"],[1528504020,"10"],[1528504080,"10"],[1528504140,"10"],[1528504200,"10"],[1528504260,"10"],[1528504320,"10"],[1528504380,"10"],[1528504440,"10"],[1528504500,"10"],[1528504560,"10"],[1528504620,"10"],[1528504680,"10"],[1528504740,"10"],[1528504800,"10"],[1528504860,"10"],[1528504920,"10"],[1528504980,"10"],[1528505040,"10"],[1528505100,"10"],[1528505160,"10"],[1528505220,"10"],[1528505280,"10"],[1528505340,"10"],[1528505400,"10"],[1528505460,"10"],[1528505520,"10"],[1528505580,"10"],[1528505640,"10"],[1528505700,"10"],[1528505760,"10"],[1528505820,"10"],[1528505880,"10"],[1528505940,"10"],[1528506000,"10"],[1528506060,"10"],[1528506120,"10"],[1528506180,"10"],[1528506240,"10"],[1528506300,"10"],[1528506360,"10"],[1528506420,"10"],[1528506480,"10"],[1528506540,"10"],[1528506600,"10"],[1528506660,"10"],[1528506720,"10"],[1528506780,"10"],[1528506840,"10"],[1528506900,"10"],[1528506960,"10"],[1528507020,"10"],[1528507080,"10"],[1528507140,"10"],[1528507200,"10"],[1528507260,"10"],[1528507320,"10"],[1528507380,"10"],[1528507440,"10"],[1528507500,"10"],[1528507560,"10"],[1528507620,"10"],[1528507680,"10"],[1528507740,"10"],[1528507800,"10"],[1528507860,"10"],[1528507920,"10"],[1528507980,"10"],[1528508040,"10"],[1528508100,"10"],[1528508160,"10"],[1528508220,"10"],[1528508280,"10"],[1528508340,"10"],[1528508400,"10"],[1528508460,"10"],[1528508520,"10"],[1528508580,"10"],[1528508640,"10"],[1528508700,"10"],[1528508760,"10"],[1528508820,"10"],[1528508880,"10"],[1528508940,"10"],[1528509000,"10"],[1528509060,"10"],[1528509120,"10"],[1528509180,"10"],[1528509240,"10"],[1528509300,"10"],[1528509360,"10"],[1528509420,"10"],[1528509480,"10"],[1528509540,"10"],[1528509600,"10"],[1528509660,"10"],[1528509720,"10"],[1528509780,"10"],[1528509840,"10"],[1528509900,"10"],[1528509960,"10"],[1528510020,"10"],[1528510080,"10"],[1528510140,"10"],[1528510200,"10"],[1528510260,"10"],[1528510320,"10"],[1528510380,"10"],[1528510440,"10"],[1528510500,"10"],[1528510560,"10"],[1528510620,"10"],[1528510680,"10"],[1528510740,"10"],[1528510800,"10"],[1528510860,"10"],[1528510920,"10"],[1528510980,"10"],[1528511040,"10"],[1528511100,"10"],[1528511160,"10"],[1528511220,"10"],[1528511280,"10"],[1528511340,"10"],[1528511400,"10"],[1528511460,"10"],[1528511520,"10"],[1528511580,"10"],[1528511640,"10"],[1528511700,"10"],[1528511760,"10"],[1528511820,"10"],[1528511880,"10"],[1528511940,"10"],[1528512000,"10"],[1528512060,"10"],[1528512120,"10"],[1528512180,"10"],[1528512240,"10"],[1528512300,"10"],[1528512360,"10"],[1528512420,"10"],[1528512480,"10"],[1528512540,"10"],[1528512600,"10"],[1528512660,"10"],[1528512720,"10"],[1528512780,"10"],[1528512840,"10"],[1528512900,"10"],[1528512960,"10"],[1528513020,"10"],[1528513080,"10"],[1528513140,"10"],[1528513200,"10"],[1528513260,"10"],[1528513320,"10"],[1528513380,"10"],[1528513440,"10"],[1528513500,"10"],[1528513560,"10"],[1528513620,"10"],[1528513680,"10"],[1528513740,"10"],[1528513800,"10"],[1528513860,"10"],[1528513920,"10"],[1528513980,"10"],[1528514040,"10"],[1528514100,"10"],[1528514160,"10"],[1528514220,"10"],[1528514280,"10"],[1528514340,"10"],[1528514400,"10"],[1528514460,"10"],[1528514520,"10"],[1528514580,"10"],[1528514640,"10"],[1528514700,"10"],[1528514760,"10"],[1528514820,"10"],[1528514880,"10"],[1528514940,"10"],[1528515000,"10"],[1528515060,"10"],[1528515120,"10"],[1528515180,"10"],[1528515240,"10"],[1528515300,"10"],[1528515360,"10"],[1528515420,"10"],[1528515480,"10"],[1528515540,"10"],[1528515600,"10"],[1528515660,"10"],[1528515720,"10"],[1528515780,"10"],[1528515840,"10"],[1528515900,"10"],[1528515960,"10"],[1528516020,"10"],[1528516080,"10"],[1528516140,"10"],[1528516200,"10"],[1528516260,"10"],[1528516320,"10"],[1528516380,"10"],[1528516440,"10"],[1528516500,"10"],[1528516560,"10"],[1528516620,"10"],[1528516680,"10"],[1528516740,"10"],[1528516800,"10"],[1528516860,"10"],[1528516920,"10"],[1528516980,"10"],[1528517040,"10"],[1528517100,"10"],[1528517160,"10"],[1528517220,"10"],[1528517280,"10"],[1528517340,"10"],[1528517400,"10"],[1528517460,"10"],[1528517520,"10"],[1528517580,"10"],[1528517640,"10"],[1528517700,"10"],[1528517760,"10"],[1528517820,"10"],[1528517880,"10"],[1528517940,"10"],[1528518000,"10"],[1528518060,"10"],[1528518120,"10"],[1528518180,"10"],[1528518240,"10"],[1528518300,"10"],[1528518360,"10"],[1528518420,"10"],[1528518480,"10"],[1528518540,"10"],[1528518600,"10"],[1528518660,"10"],[1528518720,"10"],[1528518780,"10"],[1528518840,"10"],[1528518900,"10"],[1528518960,"10"],[1528519020,"10"],[1528519080,"10"],[1528519140,"10"],[1528519200,"10"],[1528519260,"10"],[1528519320,"10"],[1528519380,"10"],[1528519440,"10"],[1528519500,"10"],[1528519560,"10"],[1528519620,"10"],[1528519680,"10"],[1528519740,"10"],[1528519800,"10"],[1528519860,"10"],[1528519920,"10"],[1528519980,"10"],[1528520040,"10"],[1528520100,"10"],[1528520160,"10"],[1528520220,"10"],[1528520280,"10"],[1528520340,"10"],[1528520400,"10"],[1528520460,"10"],[1528520520,"10"],[1528520580,"10"],[1528520640,"10"],[1528520700,"10"],[1528520760,"10"],[1528520820,"10"],[1528520880,"10"],[1528520940,"10"],[1528521000,"10"],[1528521060,"10"],[1528521120,"10"],[1528521180,"10"],[1528521240,"10"],[1528521300,"10"],[1528521360,"10"],[1528521420,"10"],[1528521480,"10"],[1528521540,"10"],[1528521600,"10"],[1528521660,"10"],[1528521720,"10"],[1528521780,"10"],[1528521840,"10"],[1528521900,"10"],[1528521960,"10"],[1528522020,"10"],[1528522080,"10"],[1528522140,"10"],[1528522200,"10"],[1528522260,"10"],[1528522320,"10"],[1528522380,"10"],[1528522440,"10"],[1528522500,"10"],[1528522560,"10"],[1528522620,"10"],[1528522680,"10"],[1528522740,"10"],[1528522800,"10"],[1528522860,"10"],[1528522920,"10"],[1528522980,"10"],[1528523040,"10"],[1528523100,"10"],[1528523160,"10"],[1528523220,"10"],[1528523280,"10"],[1528523340,"10"],[1528523400,"10"],[1528523460,"10"],[1528523520,"10"],[1528523580,"10"],[1528523640,"10"],[1528523700,"10"],[1528523760,"10"],[1528523820,"10"],[1528523880,"10"],[1528523940,"10"],[1528524000,"10"],[1528524060,"10"],[1528524120,"10"],[1528524180,"10"],[1528524240,"10"],[1528524300,"10"],[1528524360,"10"],[1528524420,"10"],[1528524480,"10"],[1528524540,"10"],[1528524600,"10"],[1528524660,"10"],[1528524720,"10"],[1528524780,"10"],[1528524840,"10"],[1528524900,"10"],[1528524960,"10"],[1528525020,"10"],[1528525080,"10"],[1528525140,"10"],[1528525200,"10"],[1528525260,"10"],[1528525320,"10"],[1528525380,"10"],[1528525440,"10"],[1528525500,"10"],[1528525560,"10"],[1528525620,"10"],[1528525680,"10"],[1528525740,"10"],[1528525800,"10"],[1528525860,"10"],[1528525920,"10"],[1528525980,"10"],[1528526040,"10"],[1528526100,"10"],[1528526160,"10"],[1528526220,"10"],[1528526280,"10"],[1528526340,"10"],[1528526400,"10"],[1528526460,"10"],[1528526520,"10"],[1528526580,"10"],[1528526640,"10"],[1528526700,"10"],[1528526760,"10"],[1528526820,"10"],[1528526880,"10"],[1528526940,"10"],[1528527000,"10"],[1528527060,"10"],[1528527120,"10"],[1528527180,"10"],[1528527240,"10"],[1528527300,"10"],[1528527360,"10"],[1528527420,"10"],[1528527480,"10"],[1528527540,"10"],[1528527600,"10"],[1528527660,"10"],[1528527720,"10"],[1528527780,"10"],[1528527840,"10"],[1528527900,"10"],[1528527960,"10"],[1528528020,"10"],[1528528080,"10"],[1528528140,"10"],[1528528200,"10"],[1528528260,"10"],[1528528320,"10"],[1528528380,"10"],[1528528440,"10"],[1528528500,"10"],[1528528560,"10"],[1528528620,"10"],[1528528680,"10"],[1528528740,"10"],[1528528800,"10"],[1528528860,"10"],[1528528920,"10"],[1528528980,"10"],[1528529040,"10"],[1528529100,"10"],[1528529160,"10"],[1528529220,"10"],[1528529280,"10"],[1528529340,"10"],[1528529400,"10"],[1528529460,"10"],[1528529520,"10"],[1528529580,"10"],[1528529640,"10"],[1528529700,"10"],[1528529760,"10"],[1528529820,"10"],[1528529880,"10"],[1528529940,"10"],[1528530000,"10"],[1528530060,"10"],[1528530120,"10"],[1528530180,"10"],[1528530240,"10"],[1528530300,"10"],[1528530360,"10"],[1528530420,"10"],[1528530480,"10"],[1528530540,"10"],[1528530600,"10"],[1528530660,"10"],[1528530720,"10"],[1528530780,"10"],[1528530840,"10"],[1528530900,"10"],[1528530960,"10"],[1528531020,"10"],[1528531080,"10"],[1528531140,"10"],[1528531200,"10"],[1528531260,"10"],[1528531320,"10"],[1528531380,"10"],[1528531440,"10"],[1528531500,"10"],[1528531560,"10"],[1528531620,"10"],[1528531680,"10"],[1528531740,"10"],[1528531800,"10"],[1528531860,"10"],[1528531920,"10"],[1528531980,"10"],[1528532040,"10"],[1528532100,"10"],[1528532160,"10"],[1528532220,"10"],[1528532280,"10"],[1528532340,"10"],[1528532400,"10"],[1528532460,"10"],[1528532520,"10"],[1528532580,"10"],[1528532640,"10"],[1528532700,"10"],[1528532760,"10"],[1528532820,"10"],[1528532880,"10"],[1528532940,"10"],[1528533000,"10"],[1528533060,"10"],[1528533120,"10"],[1528533180,"10"],[1528533240,"10"],[1528533300,"10"],[1528533360,"10"],[1528533420,"10"],[1528533480,"10"],[1528533540,"10"],[1528533600,"10"],[1528533660,"10"],[1528533720,"10"],[1528533780,"10"],[1528533840,"10"],[1528533900,"10"],[1528533960,"10"],[1528534020,"10"],[1528534080,"10"],[1528534140,"10"],[1528534200,"10"],[1528534260,"10"],[1528534320,"10"],[1528534380,"10"],[1528534440,"10"],[1528534500,"10"],[1528534560,"10"],[1528534620,"10"],[1528534680,"10"],[1528534740,"10"],[1528534800,"10"],[1528534860,"10"],[1528534920,"10"],[1528534980,"10"],[1528535040,"10"],[1528535100,"10"],[1528535160,"10"],[1528535220,"10"],[1528535280,"10"],[1528535340,"10"],[1528535400,"10"],[1528535460,"10"],[1528535520,"10"],[1528535580,"10"],[1528535640,"10"],[1528535700,"10"],[1528535760,"10"],[1528535820,"10"],[1528535880,"10"],[1528535940,"10"],[1528536000,"10"],[1528536060,"10"],[1528536120,"10"],[1528536180,"10"],[1528536240,"10"],[1528536300,"10"],[1528536360,"10"],[1528536420,"10"],[1528536480,"10"],[1528536540,"10"],[1528536600,"10"],[1528536660,"10"],[1528536720,"10"],[1528536780,"10"],[1528536840,"10"],[1528536900,"10"],[1528536960,"10"],[1528537020,"10"],[1528537080,"10"],[1528537140,"10"],[1528537200,"10"],[1528537260,"10"],[1528537320,"10"],[1528537380,"10"],[1528537440,"10"],[1528537500,"10"],[1528537560,"10"],[1528537620,"10"],[1528537680,"10"],[1528537740,"10"],[1528537800,"10"],[1528537860,"10"],[1528537920,"10"],[1528537980,"10"],[1528538040,"10"],[1528538100,"10"],[1528538160,"10"],[1528538220,"10"],[1528538280,"10"],[1528538340,"10"],[1528538400,"10"],[1528538460,"10"],[1528538520,"10"],[1528538580,"10"],[1528538640,"10"],[1528538700,"10"],[1528538760,"10"],[1528538820,"10"],[1528538880,"10"],[1528538940,"10"],[1528539000,"10"],[1528539060,"10"],[1528539120,"10"],[1528539180,"10"],[1528539240,"10"],[1528539300,"10"],[1528539360,"10"],[1528539420,"10"],[1528539480,"10"],[1528539540,"10"],[1528539600,"10"],[1528539660,"10"],[1528539720,"10"],[1528539780,"10"],[1528539840,"10"],[1528539900,"10"],[1528539960,"10"],[1528540020,"10"],[1528540080,"10"],[1528540140,"10"],[1528540200,"10"],[1528540260,"10"],[1528540320,"10"],[1528540380,"10"],[1528540440,"10"],[1528540500,"10"],[1528540560,"10"],[1528540620,"10"],[1528540680,"10"],[1528540740,"10"],[1528540800,"10"],[1528540860,"10"],[1528540920,"10"],[1528540980,"10"],[1528541040,"10"],[1528541100,"10"],[1528541160,"10"],[1528541220,"10"],[1528541280,"10"],[1528541340,"10"],[1528541400,"10"],[1528541460,"10"],[1528541520,"10"],[1528541580,"10"],[1528541640,"10"],[1528541700,"10"],[1528541760,"10"],[1528541820,"10"],[1528541880,"10"],[1528541940,"10"],[1528542000,"10"],[1528542060,"10"],[1528542120,"10"],[1528542180,"10"],[1528542240,"10"],[1528542300,"10"],[1528542360,"10"],[1528542420,"10"],[1528542480,"10"],[1528542540,"10"],[1528542600,"10"],[1528542660,"10"],[1528542720,"10"],[1528542780,"10"],[1528542840,"10"],[1528542900,"10"],[1528542960,"10"],[1528543020,"10"],[1528543080,"10"],[1528543140,"10"],[1528543200,"10"],[1528543260,"10"],[1528543320,"10"],[1528543380,"10"],[1528543440,"10"],[1528543500,"10"],[1528543560,"10"],[1528543620,"10"],[1528543680,"10"],[1528543740,"10"],[1528543800,"10"],[1528543860,"10"],[1528543920,"10"],[1528543980,"10"],[1528544040,"10"],[1528544100,"10"],[1528544160,"10"],[1528544220,"10"],[1528544280,"10"],[1528544340,"10"],[1528544400,"10"],[1528544460,"10"],[1528544520,"10"],[1528544580,"10"],[1528544640,"10"],[1528544700,"10"],[1528544760,"10"],[1528544820,"10"],[1528544880,"10"],[1528544940,"10"],[1528545000,"10"],[1528545060,"10"],[1528545120,"10"],[1528545180,"10"],[1528545240,"10"],[1528545300,"10"],[1528545360,"10"],[1528545420,"10"],[1528545480,"10"],[1528545540,"10"],[1528545600,"10"],[1528545660,"10"],[1528545720,"10"],[1528545780,"10"],[1528545840,"10"],[1528545900,"10"],[1528545960,"10"],[1528546020,"10"],[1528546080,"10"],[1528546140,"10"],[1528546200,"10"],[1528546260,"10"],[1528546320,"10"],[1528546380,"10"],[1528546440,"10"],[1528546500,"10"],[1528546560,"10"],[1528546620,"10"],[1528546680,"10"],[1528546740,"10"],[1528546800,"10"],[1528546860,"10"],[1528546920,"10"],[1528546980,"10"],[1528547040,"10"],[1528547100,"10"],[1528547160,"10"],[1528547220,"10"],[1528547280,"10"],[1528547340,"10"],[1528547400,"10"],[1528547460,"10"],[1528547520,"10"],[1528547580,"10"],[1528547640,"10"],[1528547700,"10"],[1528547760,"10"],[1528547820,"10"],[1528547880,"10"],[1528547940,"10"],[1528548000,"10"],[1528548060,"10"],[1528548120,"10"],[1528548180,"10"],[1528548240,"10"],[1528548300,"10"],[1528548360,"10"],[1528548420,"10"],[1528548480,"10"],[1528548540,"10"],[1528548600,"10"],[1528548660,"10"],[1528548720,"10"],[1528548780,"10"],[1528548840,"10"],[1528548900,"10"],[1528548960,"10"],[1528549020,"10"],[1528549080,"10"],[1528549140,"10"],[1528549200,"10"],[1528549260,"10"],[1528549320,"10"],[1528549380,"10"],[1528549440,"10"],[1528549500,"10"],[1528549560,"10"],[1528549620,"10"],[1528549680,"10"],[1528549740,"10"],[1528549800,"10"],[1528549860,"10"],[1528549920,"10"],[1528549980,"10"],[1528550040,"10"],[1528550100,"10"],[1528550160,"10"],[1528550220,"10"],[1528550280,"10"],[1528550340,"10"],[1528550400,"10"],[1528550460,"10"],[1528550520,"10"],[1528550580,"10"],[1528550640,"10"],[1528550700,"10"],[1528550760,"10"],[1528550820,"10"],[1528550880,"10"],[1528550940,"10"],[1528551000,"10"],[1528551060,"10"],[1528551120,"10"],[1528551180,"10"],[1528551240,"10"],[1528551300,"10"],[1528551360,"10"],[1528551420,"10"],[1528551480,"10"],[1528551540,"10"],[1528551600,"10"],[1528551660,"10"],[1528551720,"10"],[1528551780,"10"],[1528551840,"10"],[1528551900,"10"],[1528551960,"10"],[1528552020,"10"],[1528552080,"10"],[1528552140,"10"],[1528552200,"10"],[1528552260,"10"],[1528552320,"10"],[1528552380,"10"],[1528552440,"10"],[1528552500,"10"],[1528552560,"10"],[1528552620,"10"],[1528552680,"10"],[1528552740,"10"],[1528552800,"10"],[1528552860,"10"],[1528552920,"10"],[1528552980,"10"],[1528553040,"10"],[1528553100,"10"],[1528553160,"10"],[1528553220,"10"],[1528553280,"10"],[1528553340,"10"],[1528553400,"10"],[1528553460,"10"],[1528553520,"10"],[1528553580,"10"],[1528553640,"10"],[1528553700,"10"],[1528553760,"10"],[1528553820,"10"],[1528553880,"10"],[1528553940,"10"],[1528554000,"10"],[1528554060,"10"],[1528554120,"10"],[1528554180,"10"],[1528554240,"10"],[1528554300,"10"],[1528554360,"10"],[1528554420,"10"],[1528554480,"10"],[1528554540,"10"],[1528554600,"10"],[1528554660,"10"],[1528554720,"10"],[1528554780,"10"],[1528554840,"10"],[1528554900,"10"],[1528554960,"10"],[1528555020,"10"],[1528555080,"10"],[1528555140,"10"],[1528555200,"10"],[1528555260,"10"],[1528555320,"10"],[1528555380,"10"],[1528555440,"10"],[1528555500,"10"],[1528555560,"10"],[1528555620,"10"],[1528555680,"10"],[1528555740,"10"],[1528555800,"10"],[1528555860,"10"],[1528555920,"10"],[1528555980,"10"],[1528556040,"10"],[1528556100,"10"],[1528556160,"10"],[1528556220,"10"],[1528556280,"10"],[1528556340,"10"],[1528556400,"10"],[1528556460,"10"],[1528556520,"10"],[1528556580,"10"],[1528556640,"10"],[1528556700,"10"],[1528556760,"10"],[1528556820,"10"],[1528556880,"10"],[1528556940,"10"],[1528557000,"10"],[1528557060,"10"],[1528557120,"10"],[1528557180,"10"],[1528557240,"10"],[1528557300,"10"],[1528557360,"10"],[1528557420,"10"],[1528557480,"10"],[1528557540,"10"],[1528557600,"10"],[1528557660,"10"],[1528557720,"10"],[1528557780,"10"],[1528557840,"10"],[1528557900,"10"],[1528557960,"10"],[1528558020,"10"],[1528558080,"10"],[1528558140,"10"],[1528558200,"10"],[1528558260,"10"],[1528558320,"10"],[1528558380,"10"],[1528558440,"10"],[1528558500,"10"],[1528558560,"10"],[1528558620,"10"],[1528558680,"10"],[1528558740,"10"]]}]}}

**Comment by daniellee** [comment]
Why are your values strings? Don't think Grafana knows what to do with strings as values.

What are you using to collect the data from Kubernetes? I can recommend the [kube-state-metrics add-on](https://github.com/kubernetes/kube-state-metrics).

**Comment by torkelo** [comment]
It is how prometheus returns data, the the prometheus data source runt it through parseFloat

**Comment by torkelo** [comment]
If you click the eye you just disabled the query (which explains no data inspector or graph). 

Can you enable points in graph (display options)?

**Comment by simongun** [comment]
I activated "Draw Modes" --> "Points" under display. Same behaviour.
To summarize (I just verified the following again) ... Always accessing the Data Source (DS) in "Browser" mode

1.  if Prometheus is protected via a login session (Oauth proxy that set's a authentication session cookie) and DS is configured "With credentials"

- Grafana browser client sends cookies as expected
- Inspector get's correct response
- Response is not shown, instead, spinner

2.  if Prometheus, is not protected via login (but previously was),
2a) if DS is configured "With credentials", old cookies are sent and same behaviour as under 1)
2b) if DS is not configured "With credentials", no cookies are sent and Inspector/Graph shows correct result
